dijkstra/Workspace_Map.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
```

Comment:
if the python script is mainly a class, the filename should be in line with the class name.
Please check.